packages/cli/src/build/config.ts

```diff
@@ -172,7 +172,21 @@ export function getRollupConfigs(pkg: Package, aliases: Aliases) {
       });
     });
 
-  let hasBrowserField = pkg.entrypoints[0].json.browser !== undefined;
+  let hasBrowserField = false;
+  let hasWorkerField = false;
```

Comment:
```suggestion
  let hasWorkerCondition = false;
```

Reply:
a `*Field` made me think that we are checking `package.json#worker` here but that's not the case. In fact, the `hasBrowserField` is now slightly misleading right now too - because it's more like `hasBrowserFieldOrCondition`. Maybe worth keeping those as 2 separate variables for now? I imagine that in the future we might want to remove the support for `hasBrowserField` entirely as it's sort of a legacy feature right now - we just need to wait some time for the support for `package.json#exports` to catch up in the popular tooling (well, maybe more for wider adoption of the newest versions as the tooling, for the most part, is already ready)

Reply:
I'm not sure if the suggestion to split the browser thing into 2 variables makes sense - you'd have to check what kind of an impact it would make on the code utilizing this information

Reply:
Sure, this makes sense. I'll make the change.